### DISCUSSION TITLE
Can not update title of pull request and issue

### DISCUSSION THREAD

**Discussion by ayberkyasa** [community/community#53323] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I tried to update the title of a pull request, and an issue with the following endpoints:
`PATCH /repos/{owner}/{repo}/issues/{issue_number}`
`PATCH /repos/{owner}/{repo}/pulls/{pull_number}`
Request Body: {"title": "<NEW_TITLE>"}

I tried these endpoints in my own organization's private repositories and in my own private repositories. My PAT has the access to everything that I can select in the scope part of `Tokens (classic)`. I tried both open and closed pull requests and issues.

The request returns `200 OK`. However, the title does not change.

**Comment by fatihkaplama** [comment]
I have the same problem.
